### PR TITLE
fix(ua): change the User-Agent to use the new specs lib (version)

### PR DESCRIPTION
**Pull request by robertmogos**
change the User-Agent to use the new specs lib (version) from `JS Helper $version` to `JS Helper (${version})`

### CODE REVIEW

**Comment by Haroenv** [comment]
(shouldn't be merged before it's clarified we can deal with the new format)

File: dist/algoliasearch.helper.js [hunk 1]
```diff
@@ -13727,7 +13727,7 @@ var version = require('./version');
  */
 function AlgoliaSearchHelper(client, index, options) {
   if (client.addAlgoliaAgent && !doesClientAgentContainsHelper(client)) {
-    client.addAlgoliaAgent('JS Helper ' + version);
+    client.addAlgoliaAgent('JS Helper (' + version + ')');
```
Comment (francoischalifour) [inline dist/algoliasearch.helper.js]: The `dist/` files are generated, they're not supposed to be edited manually. You can revert these modifications.

File: src/algoliasearch.helper.js [hunk 2]
```diff
@@ -1331,7 +1331,7 @@ AlgoliaSearchHelper.prototype.clearCache = function() {
 AlgoliaSearchHelper.prototype.setClient = function(newClient) {
   if (this.client === newClient) return this;
 
-  if (newClient.addAlgoliaAgent && !doesClientAgentContainsHelper(newClient)) newClient.addAlgoliaAgent('JS Helper ' + version);
+  if (newClient.addAlgoliaAgent && !doesClientAgentContainsHelper(newClient)) newClient.addAlgoliaAgent('JS Helper (' + version + ')');
```
Comment (francoischalifour) [inline src/algoliasearch.helper.js]: Can we avoid inlining `if` statements?

Comment (robertmogos) [reply src/algoliasearch.helper.js]: I can change this but the entire lib is using this. Does it make sense to just change it in one place?

File: dist/algoliasearch.helper.js (hunk 1 above)
Comment (robertmogos) [reply dist/algoliasearch.helper.js]: Done

File: src/algoliasearch.helper.js (hunk 2 above)
Comment (francoischalifour) [reply src/algoliasearch.helper.js]: As edited in [the first UA change](https://github.com/algolia/algoliasearch-helper-js/pull/647/files#diff-cbf795d639d5f4b610b6d0f6ee395b2aR123), we don't use an inline condition. We should respect that rule for edited code IMO.

Comment (robertmogos) [reply src/algoliasearch.helper.js]: Done

**Pull request merged**